pxr/imaging/hgi/enums.h

```diff
@@ -763,6 +764,51 @@ enum HgiShaderTextureType
     HgiShaderTextureTypeArrayTexture
 };
 
+ /// \enum HgiTessellationSpacing
+ ///
+ /// Describes the type of shader tessellation spacing to use
+ ///
+ /// <ul>
+ /// <li>None:
+ ///   No tessellation spacing</li>
+ /// <li>Even:
+ ///   Even spacing. Only whole integers are considered for spacing</li>
+ /// <li>FractionalOdd:
+ ///   Fractional odd tessellation spacing</li>
+ /// <li>FractionalEven:
+ ///   Fractional even tessellation spacing</li>
+ /// </ul>
+ ///
+ enum HgiTessellationSpacing
+ {
+     HgiTessellationSpacingNone = 0,
+     HgiTessellationSpacingEven,
+     HgiTessellationSpacingFractionalOdd,
+     HgiTessellationSpacingFractionalEven
+ };
+
+ /// \enum HgiState
+ ///
+ /// Trinarian value that lets you distinguish between false and uninitialized
+ ///
+ /// <ul>
+ /// <li>Unset:
+ ///   It has not been explicitly set
+ /// <li>Diasbled:
+ ///   It has been explicitly disabled
+ /// <li>Enabled:
+ ///   It has been explicitly enabled
+ /// </ul>
+ ///
+/*
+enum HgiState {
+    Unset,
+    Disabled,
+    Enabled
+};
+ */
+typedef bool HgiState;
+
```

Comment:
Looks like this is no longer needed and we'd like to make state setting explicit.